### PR TITLE
Update language.c

### PR DESCRIPTION
**Pull request by chronoss09**

### CODE REVIEW

File: common/language.c [hunk 1]
```diff
@@ -901,7 +902,7 @@ const char* const g_strings[StrId_Max][17] =
         ),
         STR_FR(
             "Transfert…\n"
-            "%zu sur %zu Kio écrits"
+            "%zu sur %zu Ko écrits"
```
Comment (TuxSH) [inline common/language.c:905]: `Kio` is correct (2^10 bytes)

File: common/language.c [hunk 2]
```diff
@@ -848,7 +849,7 @@ const char* const g_strings[StrId_Max][17] =
             "IP Addr: %lu.%lu.%lu.%lu, Port: %d"
         ),
         STR_FR(
-            "En attente de la connexion de nxlink…\n"
+            "En attente de la connexion du nxlink…\n"
```
Comment (TuxSH) [inline common/language.c:852]: `nxlink` is a proper noun, "de" is more correct I think

File: common/language.c [hunk 3]
```diff
@@ -50,6 +50,7 @@ const char* const g_strings[StrId_Max][17] =
     {
         STR_EN("● Applet Mode ●"),
         STR_ES("● Modo Applet ●"),
+        STR_FR("● Mode applet ●"),
```
Comment (TuxSH) [inline common/language.c:53]: I wonder whether or not `Applet` should be capitalized or not, since it's a title

Comment (chronoss09) [reply common/language.c]: You have the choise

**Pull request merged**